configure

```diff
@@ -0,0 +1,338 @@
+#!/bin/sh
+### Custom configure script of Dinit.
+## Initial prepartion
+set -eu
+cd "$(dirname "$0")"
+. build/version.conf
+
+## Helper funcations
+# See http://www.etalabs.net/sh_tricks.html
```

Comment:
Don't just refer to a link (which may become invalid), include the reason in the comment.